wikidict/convert.py

```diff
@@ -312,8 +312,61 @@ def process(self) -> None:
         self.summary(file)
 
 
-def get_formaters() -> List[Type[BaseFormat]]:
-    return [KoboFormat, DFFormat]
+class StarDictFormat(DictFileFormat):
+    """Save the data into a StarDict file."""
+
+    def _convert(self) -> Path:
+        """Convert the DictFile to StarDcit."""
+        from pyglossary import Glossary
+
+        Glossary.init()
+        glos = Glossary()
+
+        # Pretty print the source
+        source = wiktionary[self.locale].format(year=date.today().year)
+        glos.setInfo("description", source)
+        glos.setInfo("title", f"Wiktionary {self.locale.upper()}-{self.locale.upper()}")
+
+        file = self.output_dir / f"dict-{self.locale}-{self.locale}.zip"
+        f = glos.convert(
+            inputFilename=file.with_suffix(".df"),
+            inputFormat="kobo_dictfile",
+            readOptions={"encoding": "utf-8"},
+            outputFilename="dict-data.ifo",
+            outputFormat="stardic",
```

Comment:
`inputFormat="Dictfile",`

`outputFormat="Stardict",`

but neither are needed since it detects by `.df` and `.ifo` extension.